### DISCUSSION TITLE
cn/2024/09/24/clt/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#182] [Announcements]
# cn/2024/09/24/clt/

https://hongtaoh.com/cn/2024/09/24/clt/

**Comment by earfanfan** [comment]
本菜鸟好奇提问，为撒用“母体”这个词语，我印象中统计学专业的课本里写的是样本与整体之类的词语。

另外，后一篇的“上古秘籍”是撒贼？其实我今年是想着把概率论与数理统计重头学一遍的，没想到被你抢先了呀。加油加油。

ps，俺国庆假期没带电脑回家，未来会断崖式失联，记得你以前说如果别人没回复会感到少许焦虑，现在应该好很多了吧。

**Reply by hongtaoh** [reply]
用「母体」是因为我不知道国内教材 population 翻译成什么。我自己感觉 母体 这个翻译还挺贴切的。

>后一篇的“上古秘籍”是撒贼

我瞎编的。

>其实我今年是想着把概率论与数理统计重头学一遍的，没想到被你抢先了呀

我之前学统计就偷懒，很多概念真的弄不明白，三年前就想把不明白的搞明白，一直没动手，现在实在是忍不住了。

>俺国庆假期没带电脑回家，未来会断崖式失联，记得你以前说如果别人没回复会感到少许焦虑，现在应该好很多了吧。

没关系的。希望你国庆愉快！

**Reply by XiangyunHuang** [reply]
国内，要是论科班的话，流传最广的应该是茆诗松的《概率论与数理统计》，按教材里的说法，母体应该翻译成总体。

**Reply by earfanfan** [reply]
我上大二时学的就是茆诗松的《概率论与数理统计》，研一的时候还又自学过一遍，然而都还给时间了。

**Comment by fenguoerbian** [comment]
小的错字，在例子的最后，是110对应的Z值，不是100对应的Z值。
另外从CLT的描述（至少你这个版本的描述）就可以看出来它需要一二阶矩存在，所以一些特殊的分布CLT是不存在的，例如柯西分布。所以那句“不论母体分布如何，……就会趋近于正态分布”并不是那么准确。
另外，提议的遵循的法则的最后一条，母体非正态且小样本，为什么t分布的估计会更好，有什么理由吗？

**Reply by hongtaoh** [reply]
业余选手居然惊动真神了。前两个确实是如此，我之前都没听过柯西分布... 谢谢指正！

最后一条，容我想想🤔

**Reply by XiangyunHuang** [reply]
@fenguoerbian 看得真仔细呀！点赞 👍
@hongtaoh 我给一个反例，[两个独立同均匀分布的随机变量之和是一个分段分布](https://xiangyun.rbind.io/2017/05/random-number-generation/)。自由度为1的t分布比标准正态分布更加矮胖，自由度越大越接近正态分布。这个矮胖就会离这个分段分布更远。

**Reply by hongtaoh** [reply]
@fengguoerbian 

>在例子的最后，是110对应的Z值，不是100对应的Z值

这个明显错误已更正。

>另外从CLT的描述（至少你这个版本的描述）就可以看出来它需要一二阶矩存在，所以一些特殊的分布CLT是不存在的，例如柯西分布。所以那句“不论母体分布如何，……就会趋近于正态分布”并不是那么准确。

确实是，研究了一番，现在知道，需要总体 (1) 均值和方差均为有限，不能无限或者不存在; (2) 样本需为独立同分布 (iid)。柯西分布的均值和方差为无限，所以其均值分布无法用中心极限定理来描述。

>提议的遵循的法则的最后一条，母体非正态且小样本，为什么t分布的估计会更好

这个是我的疏忽，总体非正态且小样本，最好通过模拟来估计样本均值分布，而非 t 分布。

@XiangyunHuang 你给的例子在这里不合适。你描述的是两个随机变量之和 (分段三角形分布)，它不涉及中心极限定理。
